src/modules/dashboard/components/Dialogs/DecisionDialog/DecisionDialog.tsx

```diff
@@ -7,37 +8,45 @@ import TextStyle from '@tiptap/extension-text-style';
 import Underline from '@tiptap/extension-underline';
 import StarterKit from '@tiptap/starter-kit';
 import * as yup from 'yup';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
 
 import Dialog, { DialogProps } from '~core/Dialog';
 import { Form } from '~core/Fields';
 import { Colony } from '~data/index';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
 
 import DialogForm from './DecisionDialogForm';
 
 const displayName = 'dashboard.DecisionDialog';
 
 export interface FormValues {
   motionDomainId: number;
-  decisionTitle?: string;
-  content?: string;
+  title: string;
+  description: string;
 }
 
 interface Props extends DialogProps {
   colony: Colony;
   ethDomainId: number;
-  decisionTitle?: string;
-  content?: string;
+  title?: string;
+  description?: string;
 }
 
 const characterLimit = 4000;
 
+export const LOCAL_STORAGE_DECISION_KEY = 'decision';
```

Comment:
Having the key as a constant means that you cannot have two decision previews open at the same time in separate tabs with different data. I would have thought that this should be possible, i.e. that each form submission receives a unique key in local storage.

Reply:
I think that you are probably misunderstanding the goal of the local storage in this instance. It is to protect a user from accidentally deleting his work (when he refreshes the page, loses internet connection etc).

What you are describing is something different and in my opinion unnecessary. Of course, it could be a feature at some point but it's not in the scope of the current one. We are not providing a platform for a user to keep draft of his thoughts/ideas, we are trying to remove the annoyance of loosing something that you already wrote.